src/alt.js

```diff
@@ -450,9 +451,14 @@ class Alt {
     }, exportObj)
   }
 
-  takeSnapshot() {
-    const state = snapshot(this)
-    this[LAST_SNAPSHOT] = state
+  takeSnapshot(...storeNames) {
+    const state = snapshot(this, ...storeNames)
+    if(this[LAST_SNAPSHOT]) {
```

Comment:
Why assign when it's a subset? I think we should always set so it's predictable.